src/Action/ExceptionAction.php

```diff
@@ -80,6 +83,12 @@ public function __invoke(FlattenException $exception, Request $request): Respons
         $headers['X-Content-Type-Options'] = 'nosniff';
         $headers['X-Frame-Options'] = 'deny';
 
-        return new Response($this->serializer->serialize($exception, $format['key']), $exception->getStatusCode(), $headers);
+        $response = new Response(null, $exception->getStatusCode(), $headers);
+
+        if (!$this->debug && $response->isServerError()) {
+            $exception->setMessage(Response::$statusTexts[$response->getStatusCode()]);
```

Comment:
So my suggestion is... why not to deal with the message in the [`ErrorNormalizer` (L40)](https://github.com/api-platform/core/blob/master/src/Problem/Serializer/ErrorNormalizer.php#L40)? And avoid the [`FlattenException::setStatusCode()`](https://github.com/api-platform/core/blob/master/src/Action/ExceptionAction.php#L71) in the same time. WDYT?